src/styles/_utils.scss

```diff
@@ -1,3 +1,4 @@
 @import "utils/vars";
 @import "utils/mixins";
 @import "utils/extends";
+@import "utils/reset";
```

Comment:
Consider placing the 'reset' import at the beginning to ensure that other styles don't get unintentionally overridden.

Reply:
I moved reset import to top.